src/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Shopify Theme Inspector for Chrome",
-  "version": "1.0.7",
+  "version": "1.0.8",
```

Comment:
We're adding a feature of knowing which rendering engine the store is on, does that classify a minor version bump? i.e `1.1.0`?